lib/ansible/modules/get_url.py

```diff
@@ -346,6 +366,17 @@ def url_filename(url):
     return fn
 
 
+def date_to_timestamp(last_mod_time):
+    # convert HTTP-date(Last-Modified) string data to timestamp
+    last_mod_timestamp = None
+    try:
+        dt = datetime.datetime.strptime(last_mod_time, "%a, %d %b %Y %H:%M:%S %Z")
+        last_mod_timestamp = time.mktime(dt.timetuple())
+    except ValueError:
+        pass
```

Comment:
Should this silently pass?  Should there be error handling, or at least a warning to let the user know this failed?

Reply:
Thanks! I have added warning message to inform covert error to users.